README.md

```diff
@@ -12,117 +13,89 @@ Generic network plugin (experimental) is designed to handle networking use cases
 - Integration with existing IPAM tools for migrating customers
 - Handle NIC's capabilities for acceleration (SRIOV/Offload/etc.)
 
-The overall design is _not_ assumed to be complete, because of ongoing work in the docker community with regards to the suitable APIs to interface with network extensions like this. Regardless, flexibility in the design has been taken into consideration to allow using a different state driver for key-value synchronization, or a different flavor of a soft-switch i.e. linux-bridge, MAC VLAN, or OpenvSwitch.
-
-The ability to specify the intent succinctly is the primary goal of the design and thus some of the specified user interface will change, and in some cases functionality will be enhanced to accommodate the same. Design details and future work is captured in a [docs/design.md](https://github.com/contiv/netplugin/blob/master/docs/Design.md).
-
-Please do not use this code in production, until code goes through more testing and few critical open issues are resolved.
-
-###Building and Testing
+The overall design is _not_ assumed to be complete, because of ongoing work in
+the docker community with regards to the suitable APIs to interface with
+network extensions like this. Regardless, flexibility in the design has been
+taken into consideration to allow using a different state driver for key-value
+synchronization, or a different flavor of a soft-switch i.e. linux-bridge, MAC
+VLAN, or OpenvSwitch.
+
+The ability to specify the intent succinctly is the primary goal of the design
+and thus some of the specified user interface will change, and in some cases
+functionality will be enhanced to accommodate the same. Design details and
+future work is captured in a
+[docs/design.md](https://github.com/contiv/netplugin/blob/master/docs/Design.md).
+
+Please do not use this code in production, until code goes through more testing
+and few critical open issues are resolved.
+
+### Getting Started
+
+This will provide you with a minimal experience of uploading the intent and
+seeing the netplugin system act on it. It will create a network on your host
+that lives behind an OVS bridge and has its own unique interfaces.
+
+```
+$ git clone https://github.com/contiv/netplugin
+$ cd netplugin; make build demo ssh
+< inside the vm >
+$ cd /opt/golang/src/github.com/contiv/netplugin
+$ sudo bin/netmaster &
+$ sudo bin/netplugin -host-label host1 &
+$ netdcli -cfg examples/one_host_multiple_nets.json
```

Comment:
This comment is not a significant one to incorporate. What you have is good, but leave an open exploration note to check out examples directory for more variations.